addons/website_event_meet/views/event_meet_templates_list.xml

```diff
@@ -48,7 +48,11 @@
             </t>
             <div t-if="not meeting_rooms" class="m-auto text-center text-muted">
                 <h3 class="mt8">No Room Open</h3>
-                <p>Create one to get conversations going</p>
+                <div groups="event.group_event_user">
+                    <a target="_blank" t-att-href="'/web?#action=website_event_meet.action_meeting_room_from_event&amp;active_id=%s&amp;model=event.track&amp;view_type=kanban' % event.id">
```

Comment:
Do you really need to specify model ? Does view_type really be really supported ?

Reply:
Yes, we don't need these actually :)

view_type and model are not specified anymore.